lib/webdriver.js

```diff
@@ -105,12 +105,17 @@ Webdriver.prototype._init = function() {
     if(err) { return cb(err); }
 
     var resData;
-    // retrieving session
+    var jsonData;
+
     try{
-      var jsonData = JSON.parse(data);
+      jsonData = JSON.parse(data);
       if( jsonData.status === 0 ){
+        // retrieving session via old JSON wire protocol
         _this.sessionID = jsonData.sessionId;
         resData = jsonData.value;
+      } else {
```

Comment:
not sure if we want a simple `else` block here. What if `status` is a field on `jsonData`, but is not `0`? What if it's some other error? In that case, are we really in W3C-land?

Reply:
We could always do something like this:

```js
if (jsonData.status === 0) {
  // old protocol, success
} else if (jsonData.value && jsonData.value.sessionId) {
  // new protocol, success
}
```
I don't think it will make a functional difference, since sessionID would wind up undefined in exceptional circumstances either way.  But perhaps using "else if" would be clearer.  Would you prefer something like that?

Reply:
Done.  Thanks!